### ISSUE TITLE
Fix SHC Logo

### ISSUE THREAD

**Issue by 27achang** [Inquiry-and-Innovation/i2Website#2]
# Issue
Current version of `/templates/student-profile.html`, Line 17:
```html
<img alt="SHCP Logo" src="/assets/SHC_Logo.png" title="SHCP Website">
```
[Line 17](https://github.com/MrStandley/i2Website/blob/main/templates/student-profile/student_profile.html#L17) of the student_profile.html template currently has the image source set to the base directory, which currently fails to render on GitHub pages.

# Solution
**Summary:** Change the reference from an absolute reference (to the base directory) to a relative reference (from the student's student profile directory)  
**Detailed Solution:** Replace `/assets/SHC_Logo.png` with `../../assets/SHC_Logo.png` so that the line reads as follows:
```html
<img alt="SHCP Logo" src="../../assets/SHC_Logo.png" title="SHCP Website">
```

**Comment by 27achang** [comment]
The i2 icon in the tab has a similar issue, detailed below.

# Issue
Current version of `/templates/student-profile.html`, Line 8:
```html
<link rel="icon" href="/assets/i2_icon_light.gif" sizes="192x192">
```
[Line 8](https://github.com/MrStandley/i2Website/blob/main/templates/student-profile/student_profile.html#L8) of the student_profile.html template currently has the image source set to the base directory, which currently fails to render on GitHub pages.

# Solution
**Summary:** Change the reference from an absolute reference (to the base directory) to a relative reference (from the student's student profile directory)  
**Detailed Solution:** Replace `/assets/i2_icon_light.gif` with `../../assets/i2_icon_light.gif` so that the line reads as follows:
```html
<link rel="icon" href="../../assets/i2_icon_light.gif" sizes="192x192">
```

**Comment by 27achang** [comment]
The template in `update-student-profile` solves this issue ([link](https://github.com/MrStandley/i2Website/blob/update-student-profile-templates/templates/student-profile/student_profile.html)).

**Comment by 27achang** [comment]
Resolved by #3
